util/util.go

```diff
@@ -302,3 +302,21 @@ func GetValue(kvPairs []string, key string) string {
 
 	return ""
 }
+
+func Map2KVPairs(m map[string]string) []string {
+	r := make([]string, 0, len(m))
+	for k, v := range m {
+		r = append(r, k + "=" + v)
+	}
+	return r
+}
+
+func KVPairs2Map(kvs []string) map[string]string {
+	r := make(map[string]string, len(kvs))
+	for _, kv := range kvs {
+		sepIdx := strings.Index(kv, "=")
```

Comment:
Thanks! Changed that. 
Go is still very new to me, so please continue pointing to more optimal ways to do stuff.